src/devices/Pigpio/Driver.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Device.Gpio;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Iot.Device.Pigpio
+{
+    /// <summary>
+    /// 
```

Comment:
Anything will do :smile:

Reply:
(not commenting on docs anymore but we require them to merge - they don't have to be exhaustive - anything reasonable is fine)